src/olympia/amo/utils.py

```diff
@@ -986,7 +985,13 @@ def utc_millesecs_from_epoch(for_datetime=None):
     """
     if not for_datetime:
         for_datetime = datetime.datetime.now()
-    return calendar.timegm(for_datetime.utctimetuple()) * 1000
+    # Number of seconds.
+    seconds = time.mktime(for_datetime.timetuple())
```

Comment:
should this be `utctimetuple` again?

Reply:
Yeah. Ultimately it doesn't matter as we only deal with UTC dates in addons-server, but we never know...